rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -370,14 +387,19 @@ public void leaveLRA(URI lraId, String body) throws WebApplicationException {
             response = client.target(coordinatorUrl)
                 .path(String.format(LEAVE_PATH, LRAConstants.getLRAUid(lraId)))
                 .request()
-                .put(Entity.text(body));
+                    .async()
```

Comment:
```suggestion
                .async()
```

Reply:
For me there is a strange formatting for these `async` calls and `get` calls. This formatting is used overall of the whole class. Is it intentional?

Reply:
Yes the whole file probably needs reformatting. I am not doing that in the PR so it can be the subject of another issue.

Reply:
Perhaps, but the formatting in this file has already diverged.